assess.dom.js

```diff
@@ -519,6 +519,25 @@ QUnit.test("duplication",
             "The two paragraphs should have the same text."
         );
 
+        assert.notEqual(
+            paras[0].id,
+            paras[1].id,
+            "Do not have the same ID for elements. " +
+            "If you want to pass along the same information/attributes then use classes instead"
+        );
+
+        assert.strictEqual(
+            paras[0].hasChildNodes(),
+            paras[1].hasChildNodes(),
+            "The two elements should have children"
+        );
+
+        assert.propEqual(
```

Comment:
I just noticed the use of `propEqual` - are you sure this does what you think it does? You might instead compare the innerHTML; then the above assertion with `hasChildNodes()` is not necessary either, or the `textContent` comparison above.